Shaded classes unavailable in trees_sjs jar versions >= 4.8.3

Shaded classes such as `scala.meta.shaded.internal.fastparse.ParserInputSource` could not be found in trees_sjs jar in versions >= 4.8.3 (but available in non-sjs/jvm trees jar):

```console
> javap -cp trees_2.13-4.8.3.jar scala.meta.shaded.internal.fastparse.ParserInputSource     
Compiled from "ParserInput.scala"
public interface scala.meta.shaded.internal.fastparse.ParserInputSource {
  public static <T> scala.meta.shaded.internal.fastparse.ParserInputSource$fromParserInput<T> fromParserInput(T, scala.Function1<T, scala.meta.shaded.internal.fastparse.ParserInput>);
  public static scala.meta.shaded.internal.fastparse.ParserInputSourceLowPri$FromReadable$ FromReadable();
  public static <T> scala.meta.shaded.internal.fastparse.ParserInputSourceLowPri$FromReadable fromReadable(T, scala.Function1<T, scala.meta.shaded.internal.geny.Readable>);
  public abstract <T> T parseThrough(scala.Function1<scala.meta.shaded.internal.fastparse.ParserInput, T>);
}
> javap -cp trees_sjs1_2.13-4.8.3.jar scala.meta.shaded.internal.fastparse.ParserInputSource
Error: class not found: scala.meta.shaded.internal.fastparse.ParserInputSource
```

This breaks some sjs compilations when upgrading to >= 4.8.3, e.g., https://jitpack.io/org/sireum/kekinian/8d2094d30f/build.log (to reproduce: https://github.com/sireum/kekinian/blob/8d2094d30ffdfe4ebe5c04646da11de647d9a17c/.jitpack.yml), log excerpt:

```log
/home/jitpack/build/slang/parser/shared/src/main/scala/org/sireum/lang/parser/SlangParser.scala:832: error: exception during macro expansion:
java.lang.ClassNotFoundException: scala.meta.shaded.internal.fastparse.ParserInputSource
	at java.base/java.net.URLClassLoader.findClass(URLClassLoader.java:445)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:588)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	at scala.meta.internal.tokenizers.ScalametaTokenizer.uncachedTokenize(ScalametaTokenizer.scala:22)
	at scala.meta.internal.tokenizers.ScalametaTokenizer.$anonfun$tokenize$1(ScalametaTokenizer.scala:17)
	at scala.collection.mutable.HashMap.getOrElseUpdate(HashMap.scala:469)
	at scala.meta.internal.tokenizers.ScalametaTokenizer.tokenize(ScalametaTokenizer.scala:17)
	at scala.meta.internal.tokenizers.ScalametaTokenizer$$anon$2.apply(ScalametaTokenizer.scala:332)
	at scala.meta.tokenizers.Api$XtensionTokenizeDialectInput.tokenize(Api.scala:25)
	at scala.meta.tokenizers.Api$XtensionTokenizeInputLike.tokenize(Api.scala:14)
	at scala.meta.internal.parsers.ScannerTokens$.apply(ScannerTokens.scala:875)
	at scala.meta.internal.parsers.ScalametaParser.<init>(ScalametaParser.scala:33)
	at scala.meta.parsers.Parse$$anon$1.apply(Parse.scala:35)
	at scala.meta.quasiquotes.Api$XtensionQuasiquoteMod$.parse(Api.scala:23)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
	at java.base/java.lang.reflect.Method.invoke(Method.java:578)
	at scala.meta.internal.quasiquotes.ReificationMacros.$anonfun$instantiateParser$2(ReificationMacros.scala:170)
	at scala.meta.internal.quasiquotes.ReificationMacros.parseSkeleton(ReificationMacros.scala:177)
	at scala.meta.internal.quasiquotes.ReificationMacros.expand(ReificationMacros.scala:60)
	at scala.meta.internal.quasiquotes.ReificationMacros.unapply(ReificationMacros.scala:55)

      case mod"override" =>
```

--- a/build.sbt
+++ b/build.sbt
@@ -865,11 +865,11 @@ lazy val docs = project
   .enablePlugins(BuildInfoPlugin, DocusaurusPlugin)
 
 lazy val shadingSettings = Def.settings(
-  shadedModules ++= {
+  shadedDependencies ++= {
     if (!isScala211.value)
       Set(
-        "com.lihaoyi" %% "geny",
-        "com.lihaoyi" %% "fastparse"
+        "com.lihaoyi" %%% "geny" % "foo",
+        "com.lihaoyi" %%% "fastparse" % "foo"
       )
     else Set.empty
   },
@@ -894,7 +894,7 @@ def crossPlatformPublishSettings(project: sbtcrossproject.CrossProject) =
 
 def crossPlatformShading(project: sbtcrossproject.CrossProject) =
   if (shadingSettings.nonEmpty)
-    project.jvmConfigure(_.enablePlugins(ShadingPlugin).settings(shadingSettings))
+    project.enablePlugins(ShadingPlugin).settings(shadingSettings)
   else project
 
 val publishJVMSettings = platformPublishSettings(JVMPlatform)

--- a/project/plugins.sbt
+++ b/project/plugins.sbt
@@ -1,3 +1,5 @@
+resolvers ++= Resolver.sonatypeOssRepos("snapshots")
+
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "2.1.1")
 
 addSbtPlugin("com.github.sbt" % "sbt-unidoc" % "0.5.0")
@@ -31,4 +33,4 @@ addSbtPlugin("org.scala-native" % "sbt-scala-native" % "0.4.14")
 
 addSbtPlugin("io.chrisdavenport" %% "sbt-npm-package" % "0.1.2")
 
-addSbtPlugin("io.get-coursier" % "sbt-shading" % "2.1.1")
+addSbtPlugin("io.get-coursier" % "sbt-shading" % "2.1.3+9-9143b95e-SNAPSHOT")

